src/topology/uniform_space/cauchy.lean

```diff
@@ -168,6 +168,12 @@ theorem cauchy_seq_tendsto_of_complete [inhabited β] [semilattice_sup β] [comp
   {u : β → α} (H : cauchy_seq u) : ∃x, tendsto u at_top (nhds x) :=
 complete_space.complete H
 
+/-- If `K` is a complete subset, then any cauchy sequence in `K` converges to a point in `K` -/
+lemma cauchy_seq_tendsto_of_is_complete [inhabited β] [semilattice_sup β] {K : set α} (h₁ : is_complete K)
```

Comment:
Hmm... shouldn't this be named something like `exists_tendsto_of_cauchy_seq_of_is_complete`, and shouldn't the previous theorem be `exists_tendsto_of_cauchy_seq`?

Actually, it would be more accurate to name it `exists_mem_tendsto_...`. This sounds pretty bad, and asking users to guess `cauchy_seq` as a prefix is reasonable. So, I'll merge in a day or two if nobody has a strong opinion about this.